package/Shaders/Lighting.hlsl

```diff
@@ -2163,7 +2167,7 @@ PS_OUTPUT main(PS_INPUT input, bool frontFace
 			[branch] if (complexMaterialParallax)
 				parallaxShadow = ExtendedMaterials::GetParallaxSoftShadowMultiplier(uv, mipLevel, lightDirectionTS, sh0, TexEnvMaskSampler, SampEnvMaskSampler, 3, parallaxShadowQuality, screenNoise, displacementParams);
 #				elif defined(TRUE_PBR) && !defined(LODLANDSCAPE)
-			[branch] if (PBRParallax)
+			[branch] if (PBRParallax && extendedMaterialSettings.EnableShadows)
```

Comment:
Why? It's queried in outer if

Reply:
My skill issue, in that case the check should be missing somewhere else, think it didn't toggle all parallax shadows.